sample/Cargo.toml

```diff
@@ -4,4 +4,5 @@ publish = false
 edition = "2024"
 
 [dependencies]
-facet = { version = "0.1.17", path = "../facet" }
+# This crate has publish=false, so it doesn't need to specify a version.
+facet = { path = "../facet" }
```

Comment:
Totally shoehorned this in to get the codegen test to pass, happy to revert it.

Reply:
I ended up doing the same somewhere, no worries